config/target_score.txt

```diff
@@ -1,5 +1,5 @@
 # Target score for scrapper.py:
-4
+8
 
 # Target score for pipeline.py:
 0
```

Comment:
It is time to implement the second lab as well

Reply:
sorry, I won't be able to do the second job. I will do it on holidays not for evaluation